docs/index.rst

```diff
@@ -23,17 +23,6 @@ API Reference
     datacatalog_v1beta1/types
 
 
-Migration Guide
```

Comment:
@gcf-owl-bot Nooooooo!

Reply:
Mailed https://github.com/googleapis/synthtool/pull/1285 to allow reStructured Text without having to add back an `owlbot.py` file.